apps/jira/jira-app/package.json

```diff
@@ -3,27 +3,28 @@
   "version": "1.1.0",
   "private": true,
   "devDependencies": {
-    "@babel/core": "7.3.4",
-    "@babel/plugin-proposal-class-properties": "7.3.4",
-    "@babel/plugin-transform-runtime": "7.3.4",
-    "@babel/preset-env": "7.3.4",
-    "@babel/preset-react": "7.0.0",
+    "@babel/core": "7.8.4",
+    "@babel/plugin-proposal-class-properties": "7.8.3",
+    "@babel/plugin-transform-runtime": "7.8.3",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-react": "7.8.3",
     "@contentful/contentful-extension-scripts": "^0.19.1",
-    "@contentful/eslint-config-extension": "0.2.0",
+    "@contentful/eslint-config-extension": "^0.2.0",
```

Comment:
Why did you unpin the dependencies here? Especially dev dependencies can/should always be pinned to avoid issues.